src/catalog/product-price-minimum-advertised.md

```diff
@@ -3,3 +3,15 @@ title: Minimum Advertised Price
 ---
 
 Merchants are sometimes prohibited from displaying a price that is lower than the manufacturer’s suggested retail price (MSRP). Minimum Advertised Price (MAP) gives you the ability to remain in compliance with the manufacturer’s requirements while offering your customers a better price. Because requirements differ from one manufacturer to another, you can configure your store to prevent the display of your actual price on pages where it is not allowed to appear according to the terms of the manufacturer.
+
+## How does it work?
+
+The Minimum Advertised Price Magento extension adds a dedicated "Click for Price" button instead of the regular product price. Therefore, if the price in your store is below the minimum set price for that product, the price is either not displayed, or the list or market price is displayed with a strikethrough to emphasize that your price is lower. When the buyer clicks the button, "Click to find out the price" - only then will the actual price at which you are selling the product become visible.
```

Comment:
Is this an extension that is installed separately, or is it part of the standard install?

Reply:
It's a core module - magento/module-msrp - in fact modules like checkout require it, so it's not one you can remove.

However, MAP is by default disabled in Magento Admin, you have to enable it.

If text is confusing then may I suggest - The Minimum Advertised Price feature , if enabled, adds a dedicated "Click for Price"

Reply:
Thanks for clarification @BarnyShergold